Mathlib/AlgebraicGeometry/Noetherian.lean

```diff
@@ -312,7 +312,6 @@ instance {R : CommRingCat} [IsNoetherianRing R] :
 instance {R} [CommRing R] [IsNoetherianRing R] :
     IsNoetherian (Spec (.of R)) := by
   suffices IsNoetherianRing (CommRingCat.of R) by infer_instance
-  simp only [CommRingCat.coe_of]
```

Comment:
`CommRingCat.coe_of` is a lemma whose proof is `rfl`.  I wonder if this `simp only` step was there just to avoid defeq abuse...